src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -299,6 +299,11 @@ public static function collectDeprecations($outputFile)
 
     private static function hasColorSupport()
     {
+        $colorEnv = getenv('ANSI');
+        if (false !== $colorEnv) {
+            return filter_var($colorEnv, FILTER_VALIDATE_BOOLEAN);
```

Comment:
I would only support `1` and `0` as values.

Reply:
The call to `filter_var` should be changed as `FILTER_VALIDATE_BOOLEAN` supports many variations of true/false strings.